### ISSUE TITLE
Please update to 4.6.1

### ISSUE THREAD

**Issue by Guite** [components/font-awesome#29]
The latest version of font-awesome is 4.6.1.

**Comment by mrohnstock** [comment]
done, don't hesitate to create a PR for a new version.

**Comment by Guite** [comment]
okay, thank you

**Comment by ducin** [comment]
npm packages are missing:
```
  time:
   { modified: '2016-04-04T19:14:50.707Z',
     created: '2016-04-04T10:46:44.513Z',
     '4.5.0': '2016-04-04T10:46:44.513Z',
     '4.4.0': '2016-04-04T12:28:33.448Z',
     '3.0.2': '2016-04-04T12:33:29.811Z',
     '3.1.0': '2016-04-04T12:34:14.912Z',
     '3.2.0': '2016-04-04T12:34:52.961Z',
     '3.2.1': '2016-04-04T12:35:47.626Z',
     '4.0.0': '2016-04-04T12:36:52.234Z',
     '4.0.1': '2016-04-04T12:38:23.736Z',
     '4.0.2': '2016-04-04T12:38:52.125Z',
     '4.0.3': '2016-04-04T12:39:10.170Z',
     '4.1.0': '2016-04-04T12:39:42.688Z',
     '4.2.0': '2016-04-04T12:43:33.079Z',
     '4.3.0': '2016-04-04T12:44:25.787Z' },
```
will take care of that later on

**Comment by ducin** [comment]
I have published npm `4.6.0` and `4.6.1` packages.
